hazelcast/src/test/java/com/hazelcast/map/LocalMapStatsTest.java

```diff
@@ -29,6 +30,7 @@
 public class LocalMapStatsTest extends HazelcastTestSupport {
 
     @Test
+    @Ignore
```

Comment:
can you also add a comment which states the reason of using `@Ignore` for a future reviewer?